test/DGmethods/compressible_Navier_Stokes/rising_bubble-model.jl

```diff
@@ -100,7 +100,8 @@ function run(mpicomm, ArrayType,
   # -------------- Define model ---------------------------------- # 
   model = AtmosModel(FlatOrientation(),
```

Comment:
The geometry naming is a bit strange: "FlatOrientation" rather than simply "LAM" (for Limited Area Model) "Cube" or "Box". I am thinking with the point of view of a user from the atmospheric community.

Reply:
@smarras79 Noted - I think this is best addressed in a separate PR.